index.js

```diff
@@ -212,31 +275,34 @@ const argTimesTwo = null; // code here!
  *     it has a `drive` method that takes a distance as its argument, and
  *         (1) causes the odometer in the object to be increased by the distance,
  *         (2) returns the updated value of the `odometer`.
-*/
-function carMaker(/* code here */) {
-  /* code here */
+ */
+function carMaker(odometer) {
+    this.odometer = odometer;
+    this.drive = distance => this.odometer += distance;
+
+    return this;
 }
 
```

Comment:
Good job attempting the stretch however this stretch asks you to return and object, which means you should actually create the object inside the function.  The first property (or key:value pair) is odometer: <parameter name here>.  The second property (or key:value pair) is drive: <then a function that takes in distance as an argument and then adds the distance to the odometer property.  You are correct in that you need the `this` keyword.